Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FeatureTransforms"
 uuid = "8fd68953-04b8-4117-ac19-158bf6de9782"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
You don't think this is breaking?

Reply:
https://github.com/invenia/FeatureTransforms.jl/issues/49#issuecomment-797071952

Reply:
We don't seem to use `is_transformable` to restrict anything in the code so it's definitely not breaking. If it were a trait or used in a test suite it might be.